static/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+
+<link rel="stylesheet" href="../client/vendor/assets/fonts/fonts.css" media="screen">
+<link rel="stylesheet" href="../client/node_modules/normalize.css/normalize.css" media="screen">
+<link rel="stylesheet" href="./main.css" media="screen" charset="utf-8">
+
+<meta name="apple-mobile-web-app-capable" content="yes">
+<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
+<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
+
+<div role="application">
+  <main>
+
+    <header>
+        <a href="https://cozy.io" class="cozy">
+          <img src="assets/illustration-welcome.svg" alt="Welcome to Cozy cloud" />
+        </a>
+
+        <h1>Welcome Michel</h1>
+        <a href="#">You're not Michel?</a>
+    </header>
+
+    <div role="region">
```

Comment:
For my culture, what is the interest of this role compared to a simple `<section>`?